tests/fixtures/rules/require-object-destructuring/bad/example.js

```diff
@@ -0,0 +1,2 @@
+let foo = SomeThing.foo;
+let bar = SomeThing.Baz.bar;
```

Comment:
Crap. I just realized that having two bad examples in a given file doesn't actually guarantee that both 'fail' (just that one or other fails causing the file to fail).